README.md

```diff
@@ -309,5 +309,47 @@ send a notification to the contract-management notification listener, which will
 Trusted Issuers List for the consumer organisation. Also compare to the flow descriptions 
 of [Consumer registration](#consumer-registration) and [Contract management](#contract-management). 
 
-A Postman collection providing example requests can be found [here](./examples/tmf/).
 
+### Authorization and authentication
```

Comment:
Is this supposed to be a new section after "Contract Management via TMForum APIs" ?

Reply:
Yes, It would be to much for one chapter and the auth/z part can be used beyond the contract example